### DISCUSSION TITLE
Yawning-Titan COP - Dev Update, Demo, and Q&A #1

### DISCUSSION THREAD

**Discussion by ChrisMcCarthyDev** [dstl/YAWNING-TITAN#35] [Ideas]
Join the Yawning-Titan dev team for a series exciting [Yawning-Titan ](https://github.com/dstl/YAWNING-TITAN) dev updates, demos, and Q&A sessions where you can stay updated on the latest developments in Yawning-Titan! Get an inside look at what's to come, learn about new key features, and have your questions answered.

Meet the teams currently managing Yawning-Titan. Discover the roadmap for Yawning-Titan, learn about the pending v1.1.0 release, and get a sneak peek at the backlog.

Get ready for a demo of Yawning-Titan DB, `YawningTitanRun` class, and the Config Revamp, plus a first look at the GUI & Network Editor. And, don't miss the opportunity to ask questions and engage in a two-way Q&A where we want to learn about how you're using Yawning-Titan.

Join us for the first session and see for yourself why [Yawning-Titan ](https://github.com/dstl/YAWNING-TITAN) is making waves in the community!

The event is being held 10:00 - 11:30 on  Wednesday 8th February 2023. For a full agenda and to sign up to the event, go to https://www.eventbrite.com/e/529065628537.

**Comment by forrestmckee** [comment]
Will a recording be posted anywhere?

**Reply by jamesshort1** [reply]
Hello @forrestmckee. We do not have a recording, but we are working through some administration to get the slides from the COP event posted on this site.
